redisgears_core/src/lib.rs

```diff
@@ -135,9 +136,11 @@ fn check_redis_version_compatible(ctx: &Context) -> Result<(), String> {
 }
 
 /// The meta information about the gears library instance at runtime.
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
```

Comment:
I am not sure why we need all those derives

Reply:
OK, I can understand that those make sense `Debug, Clone, Eq, PartialEq` but I am not sure about `Ord, PartialOrd, Hash`. Do you see a need to order library meta data. And if so, not sure the default implementation is the right one. WDYT?

Reply:
OK, convinced :)
Lets keep it.